lib/codelog/command/step/version.rb

```diff
@@ -12,28 +12,60 @@ class Version
         UNRELEASED_LOGS_PATH = 'changelogs/unreleased'.freeze
         CONFIG_FILE_PATH = 'changelogs/codelog.yml'.freeze
 
-        def initialize(version, release_date)
+        def initialize(version, release_date, options = {})
           abort(Codelog::Message::Error.missing_config_file) unless config_file_exists?
           @version = version
           @release_date = Date.strptime(release_date, Codelog::Config.date_input_format).to_s
+          @options = options
         rescue ArgumentError
           abort(Codelog::Message::Error.invalid_date_format)
         end
 
-        def self.run(version, release_date)
-          Codelog::Command::Step::Version.new(version, release_date).run
+        def self.run(version, release_date, options = {})
+          Codelog::Command::Step::Version.new(version, release_date, options).run
         end
 
         def run
           abort(Codelog::Message::Error.missing_version_number) if @version.nil?
           abort(Codelog::Message::Error.already_existing_version(@version)) if version_exists?
           abort(Codelog::Message::Error.no_detected_changes(@version)) unless unreleased_changes?
+
+          choose_output
+        end
+
+        private
+
+        def choose_output
```

Comment:
@Guilherme2112 

I think this class [shouldn't be responsible](https://en.wikipedia.org/wiki/Single_responsibility_principle) for managing the output logic.

Maybe this class would be better off if we abstracted the output logic away and only injected an "`outputer`" here, leaving the decision about which "`outputer`" to use to the consumer classes of `Version`